plugins/modules/podman_import.py

```diff
@@ -127,29 +135,65 @@ def load(module, executable):
         module.fail_json(msg="Could not parse JSON from image %s: %s" % (image_name, e))
     return changed, out, err, info, command
 
+def volume_load(module, executable):
+    changed = False
+    command = [executable, 'volume', 'import', module.params['volume'], module.params['src']]
+    src = module.params['src']
+    if os.path.exists(src):
+        changed = True
```

Comment:
In this case it's always `True`, or module fails, I don't see an option it exits with `changed=False`. Probably it should check if current volume exists and not to do anything if it exists? I'm not sure, because Podman imports volume in any case and overrides the existing ones.

Reply:
So either it is always true, or we need to check somehow that the existing volume is the same as the tarball ?
Is there a way to do that ?

Reply:
I think it's an overkill to check it. Podman just overrides, probably we need to follow this logic (whatever it's right or not), so we'll have it always changed=true.